cmd/utils/flags.go

```diff
@@ -1206,7 +1213,28 @@ func setTxPool(ctx *cli.Context, cfg *core.TxPoolConfig) {
 	}
 }
 
-func setProgpow(ctx *cli.Context, cfg *ethconfig.Config) {
+func setBlake3DurationLimit(ctx *cli.Context, cfg *ethconfig.Config) {
```

Comment:
I don't think these functions should be duplicated. The duration limits do not change between blake3 or progpow. Just have one `setDurationLimit()` method, that assigns the duration limits for whichever consensus engine is active